pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -47,10 +47,13 @@ If this is not enough, try::
 
 or::
 
-  addAction(MaxQPSIPRule(5), TCAction())
+  addAction(AndRule{MaxQPSIPRule(5), TCPRule(false)}, TCAction())
 
 This will respectively drop traffic exceeding that 5 QPS limit per IP or range, or return it with TC=1, forcing clients to fall back to TCP.
 
+In that last one, note the use of :func:`TCPRule`.
+Without it, clients would get TC=1 even if they correctly fell back to TCP.
```

Comment:
Maybe note the 1.7.0 change here too?

Reply:
I pondered that but could not find a way not involving many words, and figured that check was cheap anyway :-)

Reply:
Fair!